engine/src/main/scala/cromwell/server/CromwellShutdown.scala

```diff
@@ -217,5 +218,13 @@ object CromwellShutdown extends GracefulStopSupport {
       logger.info("Stream materializer shut down")
       Future.successful(Done)
     }
+
+    // 7) Close out the backend used for WDL HTTP import resolution
+    // http://sttp.readthedocs.io/en/latest/backends/start_stop.html
+    coordinatedShutdown.addTask(CoordinatedShutdown.PhaseBeforeActorSystemTerminate, "wdlHttpImportResolverBackend") { () =>
+      HttpResolver.sttpBackend.close()
```

Comment:
@aednichols looks good to me :)
Although the same comment @cjllanwarne made below applies here too of course.